doc/classes/SpinBox.xml

```diff
@@ -5,6 +5,19 @@
 	</brief_description>
 	<description>
 		SpinBox is a numerical input text field. It allows entering integers and floats.
+		[b]Example:[/b]
+		[codeblock]
+		func _ready():
+		  var spin_box = SpinBox.new()
+		  get_tree().get_root().add_child(spin_box)
+		  var line_edit = $SpinBox.get_line_edit()
+		  var is_enabled = false
+		  line_edit.set_context_menu_enabled(is_enabled)
+		  var text_alignment = LineEdit.ALIGN_RIGHT
+		  spin_box.set_align(text_alignment)
```

Comment:
Same here, no need to save the value in a variable before using it, and direct property access should be favored, so:
```
spin_box.align = LineEdit.ALIGN_RIGHT
```